elastic/eui
# [EuiFormControlLayout] Fix `isInvalid` icon

## Summary

This conversion was missed in https://github.com/elastic/eui/pull/6608. I did a grep for the old `alert` type and did not see any other missed conversions.

This fix needs to be merged in and a patch release done ASAP / before the next Kibana upgrade, as form controls are some of our most used/common components.

## QA

- Go to https://elastic.github.io/eui/#/forms/form-controls and any display toggles and confirm the icon is broken when `isInvalid` is toggled
- Go to https://eui.elastic.co/pr_6629/#/forms/form-controls and confirm the icon is no longer broken when a form control's invalid display is toggled

### General checklist
